pages/search.tsx

```diff
@@ -173,35 +183,48 @@ const Search: NextPage = () => {
               </Typography>
 
               {searchResults.length > 0
-              && (
-                <ViewModeActionsContainer>
-                  <Typography variant="body_short"><FormattedMessage id="search.view" /></Typography>
-
-                  <Button variant="ghost_icon" color="secondary">
-                    <Icon data={list} />
-                  </Button>
-
-                  <Button variant="ghost_icon" color="secondary">
-                    <Icon data={gridOn} />
-                  </Button>
-                </ViewModeActionsContainer>
-              )}
+                && (
+                  <ViewModeActionsContainer>
+                    <Typography variant="body_short"><FormattedMessage id="search.view" /></Typography>
+
+                    <Button variant="ghost_icon" color="secondary">
+                      <Icon data={list} />
+                    </Button>
+
+                    <Button variant="ghost_icon" color="secondary">
+                      <Icon data={gridOn} />
+                    </Button>
+                  </ViewModeActionsContainer>
+                )}
```

Comment:
Is something wrong with the eslint setup somewhere since we get these kind of spacing issues? 🤔

Reply:
This one i am not quite sure why its happening.